### PR TITLE
feat: show user's roles and project owners

### PR DESCRIPTION
**Pull request by thomasheartman**
This change shows the user's roles and project owners in the personal dashboard.

### CODE REVIEW

File: frontend/src/openapi/models/personalDashboardSchemaProjectsItem.ts [hunk 1]
```diff
@@ -4,7 +4,32 @@
  * See `gen:api` script in package.json
  */
 
+export type SystemOwner = { ownerType: 'system' };
```
Comment (kwasniew) [inline frontend/src/openapi/models/personalDashboardSchemaProjectsItem.ts:7]: is everything here auto generated?

File: frontend/src/component/personalDashboard/RoleAndOwnerInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        <Wrapper>
+            <RoleInfo>
+                <span>Your roles in this project:</span>
+                {roles.map((role) => (
+                    <Badge key={role.id} color='secondary'>
+                        {role.name}
+                    </Badge>
+                ))}
+            </RoleInfo>
+            <RoleInfo>
```
Comment (kwasniew) [inline frontend/src/component/personalDashboard/RoleAndOwnerInfo.tsx:70]: This one is owner info, not RoleInfo. Maybe a more generic name for the wrapper?

Comment (thomasheartman) [reply frontend/src/component/personalDashboard/RoleAndOwnerInfo.tsx:70]: Yeah, perfect. Forgot about that; will do

File: frontend/src/component/personalDashboard/PersonalDashboard.tsx [hunk 3]
```diff
@@ -261,9 +270,10 @@ export const PersonalDashboard = () => {
                                                 />
                                             </IconButton>
                                         </ProjectBox>
-                                        {project.name === activeProject ? (
+                                        {project.id === activeProject?.id ? (
                                             <ActiveProjectDetails
-                                                project={project}
+                                                // @ts-ignore
```
Comment (kwasniew) [inline frontend/src/component/personalDashboard/PersonalDashboard.tsx:275]: is removal of the ignore part of the followup PR?

File: frontend/src/openapi/models/personalDashboardSchemaProjectsItem.ts (hunk 1 above)
Comment (thomasheartman) [reply frontend/src/openapi/models/personalDashboardSchemaProjectsItem.ts:7]: No. it's hand-crafted and copied. I didn't wanna generate orval just yet, but the schema is in, so it should be possible to do. lemme have a look.

File: frontend/src/component/personalDashboard/PersonalDashboard.tsx [hunk 4]
```diff
@@ -274,26 +284,24 @@ export const PersonalDashboard = () => {
                 </SpacedGridItem>
                 <SpacedGridItem item lg={4} md={1}>
                     {onboardingCompleted ? (
-                        <ProjectSetupComplete project={activeProject} />
+                        <ProjectSetupComplete project={activeProject!.id} />
```
Comment (kwasniew) [inline frontend/src/component/personalDashboard/PersonalDashboard.tsx:287]: Is there a plan to remove "!" ?

**Review by kwasniew — APPROVED** [review]

File: frontend/src/component/personalDashboard/PersonalDashboard.tsx (hunk 4 above)
Comment (thomasheartman) [reply frontend/src/component/personalDashboard/PersonalDashboard.tsx:287]: Yes, that's also one! There's some TS inference stuff going on here, I think

**Pull request merged**